core/templates/dev/head/components/collection_summary_tile_directive.html

```diff
@@ -0,0 +1,52 @@
+<script type="text/ng-template" id="summaryTile/collection">
+  <md-card class="oppia-gallery-summary-tile bottom-card">
+    <div class="top-section" style="background-color: <[getThumbnailBgColor()]>;">
+    </div>
+    <md-card class="oppia-gallery-summary-tile middle-card">
+      <div class="top-section" style="background-color: <[getThumbnailBgColor()]>;">
+      </div>
+      <md-card class="oppia-gallery-summary-tile top-card">
+        <a ng-href="<[getCollectionLink()]>">
+          <div class="top-section" style="background-color: <[getThumbnailBgColor()]>;">
+            <img class="thumbnail-image" ng-src="<[getThumbnailIconUrl()]>">
+            <h2 class="title"><[getCollectionTitle()]></h2>
+          </div>
+
+          <div>
+            <div class="objective">
+              <[getObjective() | truncateAndCapitalize: 45]>
+              <span ng-if="!getObjective()">No objective specified.</span>
+            </div>
+
+            <ul layout="row" class="metrics" layout-align="space-between center">
+              <li>
+                <span>
+                  COLLECTIONS(<[getNumExplorations()]>)
+                </span>
+              </li>
+
+              <li>
+                <span ng-if="!wasRecentlyUpdated()">
+                  <[getLastUpdatedDatetime()]>
+                </span>
+                <span ng-if="wasRecentlyUpdated()" class="recently-updated">
+                  UPDATED
+                </span>
+              </li>
+            </ul>
+          </div>
+        </a>
+      </md-card>
+    </md-card>
+  </md-card>
+</script>
+
+<style>
+  .oppia-gallery-summary-tile.middle-card, .oppia-gallery-summary-tile.top-card {
```

Comment:
When playing around with this, I ended up adding a small negative-y box shadow to the middle and top cards. It helped better distinguish the cards are being stacked, since it's sort of hard to see right now. What do you think?

I used:

``box-shadow: 0px -1px 5px 0 rgba(0, 0, 0, 0.26);`` though perhaps using a smaller radius would be better.